source/particle/property/integrated_strain.cc

```diff
@@ -43,33 +44,35 @@ namespace aspect
                                                           const std::vector<Tensor<1,dim> > &gradients,
                                                           std::vector<double> &data) const
       {
-        SymmetricTensor<2,dim> old_strain;
-        for (unsigned int i = 0; i < SymmetricTensor<2,dim>::n_independent_components ; ++i)
-          old_strain[SymmetricTensor<2,dim>::unrolled_to_component_indices(i)] = data[data_position + i];
+        Tensor<2,dim> old_strain;
+        for (unsigned int i = 0; i < Tensor<2,dim>::n_independent_components ; ++i)
+          old_strain[Tensor<2,dim>::unrolled_to_component_indices(i)] = data[data_position + i];
 
         Tensor<2,dim> grad_u;
         for (unsigned int d=0; d<dim; ++d)
           grad_u[d] = gradients[d];
 
         const double dt = this->get_timestep();
 
-        // Here we update the integrated strain by rotating the already existing
-        // strain with the rotational (asymmetric) part of the velocity gradient
-        // tensor, and afterwards add the additionally experienced deformation.
-        const SymmetricTensor<2,dim> deformation_rate (symmetrize(grad_u));
+        Tensor<2,dim> new_strain;
 
-        // rotation tensor =
-        //     asymmetric part of the displacement in this time step
-        //                (= time step * (velocity gradient tensor - symmetric_part))
-        //   + unit tensor
-        const Tensor<2,dim> rotation (dt * (grad_u - deformation_rate) + unit_symmetric_tensor<dim>());
+        const Tensor<2,dim> k1 = grad_u * old_strain * dt;
```

Comment:
can you add a comment before this block that (i) states the ODE you are integrating, and (ii) just says that you are using an RK4 integrator?